content/en/_index.html

```diff
@@ -93,7 +93,9 @@ <h5 class="card-title text-white section-head">Model serving</h5>
             TensorFlow models to Kubernetes. Kubeflow is also integrated with
             <a target="_blank" rel="noopener" href="/docs/external-add-ons/serving/seldon/">Seldon Core</a>, an open source platform for deploying machine learning
             models on Kubernetes, <a target="_blank" rel="noopener" href="/docs/external-add-ons/serving/tritoninferenceserver/">NVIDIA Triton Inference Server</a> for
-            maximized GPU utilization when deploying ML/DL models at scale, and <a target="_blank" rel="noopener" href="/docs/external-add-ons/serving/mlrun/">MLRun Serving</a>, an open-source serverless framework for deployment and monitoring of real-time ML/DL pipelines.
+            maximized GPU utilization when deploying ML/DL models at scale, <a target="_blank" rel="noopener" href="/">BentoML</a>, an open-source platform for building,
```

Comment:
@yubozhao how is BentoML integrated with `Model serving` in Kubeflow? Is this via KServe? Is there some documentation for it?

Reply:
@yubozhao I agree with Kimonas' questions above.    To post this update, Bento should have some docs on how users can execute a workflow from Kubeflow to Bento.